app/pibakery.js

```diff
@@ -70,11 +70,12 @@ var tempBlocks = []
   * @return null
 */
 function getOsPath () {
-  if (process.platform == 'darwin') {
+  var platform = process.platform;
+  if ((platform == 'darwin') || (platform == 'linux') || (platform == 'freebsd')) {
     // Mac stores the OS in Application Support
     return path.normalize('/Library/Application Support/PiBakery/os/')
```

Comment:
I'm actually also working on the linux version, and it needs a fee more changes than what's in this pull request. As for where the os folder should be stored, it'll be OK to have it in the folder where the source is (like on Windows). The reason Mac has it outside of that and in `Application Support` is because there seemed to be an issue opening an .app that was larger than 1GB.